pages/common/rustup-default.md

```diff
@@ -0,0 +1,8 @@
+# rustup default
+
+> Set the default Rust toolchain.
+> More information: <https://rust-lang.github.io/rustup>.
+
+- Switch the default Rust toolchain (see `rustup help toolchain` for more information):
+
+`rustup default {{toolchain}}`
```

Comment:
In my opinion, can we add a concrete example here ?
e.g `nightly|beta|stable` ?

```suggestion
`rustup default {{nightly|beta|stable}}`
```

Reply:
We can, but there's many more things that can be put there.

`rustup toolchain help`
```
standard release channel toolchain names have the following form:

        <channel>[-<date>][-<host>]

        <channel>       = stable|beta|nightly|<major.minor>|<major.minor.patch>
        <date>          = YYYY-MM-DD
        <host>          = <target-triple>

'channel' is a named release channel, a major and minor version
number such as `1.42`, or a fully specified version number, such
as `1.42.0`. Channel names can be optionally appended with an
archive date, as in `nightly-2014-12-18`, in which case the
toolchain is downloaded from the archive for that date.
```

Reply:
Sure that makes sense.
I will leave it to your `taste & judgment` :)
Thanks :)